modin/backends/pandas/query_compiler.py

```diff
@@ -507,6 +507,25 @@ def transpose(self, *args, **kwargs):
         # Switch the index and columns and transpose the data within the blocks.
         return self.__constructor__(self._modin_frame.transpose())
 
+    def columnarize(self):
+        """
+        Transposes this QueryCompiler if it has a single row but multiple columns.
+
+        This method should be called for QueryCompilers representing a Series object,
+        i.e. self.is_series() should be True.
+
+        Returns:
+            Transposed new QueryCompiler or self.
+        """
+        if len(self.columns) != 1 or (
+            len(self.index) == 1 and self.index[0] == "__reduced__"
+        ):
+            return self.transpose()
+        return self
+
+    def is_series(self):
+        return len(self.columns) == 1 or len(self.index) == 1
```

Comment:
Since `is_series` may be called from any query compiler that contains either `DataFrame` or `Series`, I think the condition should be changed as follows.
```suggestion
        return (len(self.columns) == 1 and self.columns[0] == "__reduced__") or (len(self.index) == 1 and self.index[0] == "__reduced__")
```

Reply:
IIUC `"__reduced__"` column name is used for unnamed series only, i.e. when its name is `None`. For named series column name would hold the original series name. Anyway, this function was introduced to check the shape only. E.g. in `DataFrame` this can be used in `squeeze` method. Probably, we can change method name to `is_series_like` or smth like that.

Reply:
Yes, you are right regarding to `__reduced__`. If this function assumes to be used for check of shape only, we could to rename it to `is_series_like` and add some doc string on what does function do. What do you think?

Reply:
Sounds good, will fix that.